crates/bevy_ecs/src/component.rs

```diff
@@ -153,40 +153,12 @@ use std::{
 pub trait Component: Send + Sync + 'static {
     /// A marker type indicating the storage type used for this component.
     /// This must be either [`TableStorage`] or [`SparseStorage`].
```

Comment:
```suggestion
    /// This must be either [`StorageType::Table`] or [`StorageType::SparseSet`].
```
Or just remove this line since it's an enum now.

Reply:
Yeah we should just remove it.
```suggestion
```